containers/hub.Containerfile

```diff
@@ -1,20 +1,33 @@
 # Builds should be available on https://quay.io/organization/openscanhub-fedora-infra/ocp
-FROM registry.access.redhat.com/ubi9/httpd-24
+# TODO: Check if we can use ubi.
+# resalloc dependencies require a subscription.
+# FROM registry.access.redhat.com/ubi9/httpd-24
+FROM quay.io/sclorg/httpd-24-c9s
```

Comment:
yeah, is stream an option here too?

Reply:
This container image is based on `c9s`, although it has not been updated in last couple of years. I have requested sclorg team to start rebuilding it. I would resolve this comment when it is updated.

Reply:
https://github.com/sclorg/httpd-container/pull/203

Reply:
The pull request was merged today and the image has been updated on [quay.io](https://quay.io/repository/sclorg/httpd-24-c9s?tab=history). Thanks @zmiklank!